index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Skeleton project</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+
+  <body>
+    <header>
+      <section class="navigator1">
+        <div class="navigator">
+          <span class="mylogo">My logo</span>
+          <span><img src="images/Enabled.png" alt="" class="hamburger"></span>
+        </div>
+
+      </section>
+    </header>
+
+    <main class="container">
+      <section class="container1">
+        <h1 class="head">I’m Suzana <br> Glad to see you!</h1>
+        <p class="pr">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#">LET’S CONNECT</a>
```

Comment:
- [ ] Please try to match it like the design in Figma and It is not required to be <a> tag if you need to change it.